index.html

```diff
@@ -8,10 +8,16 @@
     <title>Jamazon</title>
   </head>
   <body>
-    <h1></h1>
+    <div id="cart">
+      <a href="#">Cart <span class="badge">(0)</span></a>
+    </div>
+    <div>
+      <h1></h1>
+    </div>
     <div id="app">
       <div data-view="catalog"></div>
       <div data-view="details"></div>
+      <div data-view="cart"></div>
```

Comment:
We'll be dealing with the cart view in a couple issues. You can remove this `data-view` for now.